client/dashboard/sites/settings-transfer-site/confirm-new-owner-form.tsx

```diff
@@ -0,0 +1,85 @@
+import { DataForm, isItemValid } from '@automattic/dataviews';
+import {
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	Button,
+} from '@wordpress/components';
+import { createInterpolateElement } from '@wordpress/element';
+import { sprintf, __ } from '@wordpress/i18n';
+import { useState } from 'react';
+import type { Field } from '@automattic/dataviews';
+
+type ConfirmNewOwnerFormData = {
+	email: string;
+};
+
+const fields: Field< ConfirmNewOwnerFormData >[] = [
+	{
+		id: 'email',
+		label: __( 'Email' ),
+		type: 'email' as const,
+	},
+];
+
+const form = {
+	type: 'regular' as const,
+	fields: [ 'email' ],
+};
+
+export function ConfirmNewOwnerForm( {
+	siteSlug,
+	handleSubmit,
+}: {
+	siteSlug: string;
+	handleSubmit: ( event: React.FormEvent ) => void;
+} ) {
+	const [ formData, setFormData ] = useState( {
+		email: '',
+	} );
+
+	const isSaveDisabled = ! isItemValid( formData, fields, form );
+
+	return (
+		<VStack spacing={ 1 }>
+			<VStack style={ { padding: '8px 0' } }>
+				<Text size="15px" weight={ 500 } lineHeight="32px">
```

Comment:
It seems weird to have to specify inline styles like this... is this not a `<Heading />` or something? 🥲

Reply:
I remembered the heading doesn't have any level with 15px 🤔

Reply:
I think we should raise it with the designers... it's very easy to have inconsistent headings if this is not standardized 🤔